include/Dialect/Poly/IR/PolyTypes.td

```diff
@@ -20,8 +20,24 @@ def Polynomial : Poly_Type<"Poly", "poly"> {
     A type for polynomials in a polynomial quotient ring.
   }];
 
-  let parameters = (ins Ring_Attr:$ring);
-  let assemblyFormat = "`<` $ring `>`";
+  let parameters = (ins
+    Ring_Attr:$ring,
+    OptionalParameter<"Attribute">:$encoding
+  );
+
+  // Skip the default builder since the custom builder enables the default
+  // value for the encoding parameter, And I couldn't get DefaultValueParameter
+  // to auto-generate a builder with the appropriate default value.
```

Comment:
if you file a follow-up issue for this i'm happy to take a look - i was messing around with default values at some point.

Reply:
Done